### PR TITLE
Increase max-width to 1500; stripe table

### PR DESCRIPTION
**Pull request by brandones**
Screenshot from 2020-04-09 18-56-07

### CODE REVIEW

File: src/referrals-queue/referrals-queue.css [hunk 1]
```diff
@@ -44,13 +44,11 @@
   margin: 2rem 2rem 1rem 0;
 }
 
-/* Copied from Sancho UI */
 .dropdown {
   width: 100%;
   height: 3.5rem;
   background: transparent;
   border: 1px solid var(--omrs-color-ink-lowest-contrast);
-  border-radius: 0.25rem;
```
Comment (brandones) [inline src/referrals-queue/referrals-queue.css:53]: This was redundant

File: src/referrals-queue/referrals-queue.css [hunk 2]
```diff
@@ -44,13 +44,11 @@
   margin: 2rem 2rem 1rem 0;
 }
 
-/* Copied from Sancho UI */
```
Comment (brandones) [inline src/referrals-queue/referrals-queue.css:47]: At this point, "loosely inspired" would be more accurate

**Review by mogoodrich — APPROVED** [review]
LGTM... we should test this on different screen sized, specifically the one they have at Mirebalais on the thin clients.

**Comment by brandones** [comment]
@mogoodrich How wide is the thin client?

**Pull request merged**